e2e/app/page/authenticated-page.ts

```diff
@@ -33,14 +31,8 @@ export default abstract class AuthenticatedPage extends BasePage {
    * Wait until current page is loaded and without spinners spinning.
    */
   async waitForLoad(): Promise<this> {
-    try {
-      await this.isSignedIn();
-      await this.isLoaded();
-    } catch (err) {
-      await savePageToFile(this.page);
-      await takeScreenshot(this.page);
-      throw (err);
```

Comment:
redundant. saving screenshot and html is done in 
[handleTestEvent](https://github.com/all-of-us/workbench/blob/e7b992d15bf2a72cec3520f72a74a8e3aab42a60/e2e/puppeteer-custom-environment.ts#L30)